processing/src/main/java/io/druid/segment/virtual/SingleLongInputCachingExpressionColumnValueSelector.java

```diff
@@ -143,4 +143,10 @@ private ExprEval eval(final long value)
     bindings.set(value);
```

Comment:
Checks in this class in getDouble/Long?

@nishantmonu51 please review once again ALL classes that implement `ColumnValueSelector` in the Druid project, without skipping those which extend each other, if they need those checks or not.

Reply:
I intentionally skipped checks for expression selectors, as all subclasses of ExprEval already have checks.

Reply:
Please add comments for readers, expecting asserts

Reply:
added.